parl/remote/message.py

```diff
@@ -0,0 +1,19 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CONNECT_TAG = b'[CONNECT]'
```

Comment:
Importing a lot global variables is not a user-friendly way, reducing the readability.

Reply:
Now, will import remote_constants and use like `remote_constants.CONNECT_TAG`